#161348789 (Integrate Coveralls):Integrate Coveralls code coverage service

dnuwa opened the pull request:
#### Description of Task to be completed?
Integrate Coveralls code coverage service
### How should this be manually tested?
the badge should show build status
#### What are the relevant pivotal tracker stories?
#161348787

Commit:
chore(Integrate Coveralls code):
-Add a coveralls badge to the README.md file
-Add a coveralls.yml  with a token to the project repo
[Fixes #161348787]

emugaya commented on .coveralls.yml:
```diff
@@ -0,0 +1 @@
+repo_token: yaikfkaACZ9SpRfRlvqw6Z76oPmC907ZF
```
Add new line

emugaya replied:
@dnuwa set this in environment variable

dnuwa replied:
The coveralls.yml file was deleted because upon doing research it is not needed due to the fact that the application uses the travis.yml file to take care of the test coverage settings.

brucemakallan commented:
Please add the ID of the Chore from Pivotal Tracker (to the heading of the PR).

Commit:
chore(Integrate Coveralls):Integrate Coveralls code coverage service
- edit coveralls file to add new line
[Finishes #161348789]

Commit:
chore(Integrate Coveralls):Integrate Coveralls code coverage service
- remove .coveralls.yml, not required as long as Travis Ci builds correctly
[Finishes #161348789]
